mycroft/tts/tts.py

```diff
@@ -464,6 +465,10 @@ def get_tts_class(self):
         pass
 
 
+def load_tts_plugin(module_name):
```

Comment:
what is the purpose of this helper function? it is only executed once.

Reply:
For keeping the code below more readable, but I'll see if I can think of another way to keep it clear.